src/providers/workers-ai/chatComplete.ts

```diff
@@ -0,0 +1,188 @@
+import { WORKERS_AI } from '../../globals';
+import { Params, Message } from '../../types/requestBody';
+import {
+  ChatCompletionResponse,
+  ErrorResponse,
+  ProviderConfig,
+} from '../types';
+import { generateErrorResponse, generateInvalidProviderResponseError } from '../utils';
+
+export const WorkersAiChatCompleteConfig: ProviderConfig = {
+  messages: [
```

Comment:
As per their docs, workersAI also supports messages input. So there is ideally no need to make transformation for chat complete calls.

Here is their docs link which has info on this: https://developers.cloudflare.com/workers-ai/models/sqlcoder-7b-2/#scoped-prompts

Reply:
@VisargD good catch, that was a lazy copy/paste from another provider...

Fixed in https://github.com/Portkey-AI/gateway/pull/269/commits/b0fae5f4d28d761b56580fc3c5d162f94029cd8b